openbas-api/src/main/java/io/openbas/rest/scenario/form/InjectsImportTestInput.java

```diff
@@ -0,0 +1,25 @@
+package io.openbas.rest.scenario.form;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.openbas.rest.mapper.form.ImportMapperAddInput;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+import static io.openbas.config.AppConfig.MANDATORY_MESSAGE;
+
+@Data
+public class InjectsImportTestInput {
```

Comment:
Why this name ?
Why we don't keep the nomenclature XlsImportMapperTestInput

Reply:
The previous POJO in io/openbas/rest/scenario/form was InjectsImportInput so we're keeping the nomenclature used for the other ones here (or should we rename them both ?)

Reply:
I think I'm having difficulty with the whole nomenclature
We can talk about it in person